Refactor style-only HMR cache and performance (#9712)
withastro/astro: .changeset/eight-turtles-itch.md

---
"astro": patch
---

> Improves Astro file style-only HMR caching and performance

Reviewer: ```suggestion
Improves HMR behavior for style-only changes in `.astro` files
```
